### DISCUSSION TITLE
Spore Meeting  Agenda @ 2024-01-30

### DISCUSSION THREAD

**Discussion by ashuralyk** [sporeprotocol/spore-scrum#20] [Show and tell]
# Goal

[[previous]](https://github.com/sporeprotocol/spore-scrum/discussions/17)

1. cooperate with Philosopher Stone to complete its adoption by BuilderDAO
2. think out our future plan to our next vision in NFT industry
3. finish work rebalance after breaking change in team

talks: @RongxinZhang 
* combine Spore and Scutt to one team?
* routine marketing continue?
* remove `spore-demo` from our doc-site?
* discuss again about our transparency to community.

# Previous Work

1. `spore-contract` successfully deployed on CKB mainnet with a proper tag
    - [x] https://github.com/sporeprotocol/spore-contract/pull/52
    - [x] https://github.com/sporeprotocol/spore-contract/pull/55

2. `spore-sdk` adapted to the Spore contract mainnet migration
    - [x] https://github.com/sporeprotocol/spore-sdk/pull/79
    - [x] https://github.com/sporeprotocol/spore-sdk/pull/82

# Next Plan

1. write out a paper to describe a perspective on our future direction
2. take over the work from `spore-sdk` and `spore-graphql`
3. determine our Devrel's next plan

------
@ShookLyngs @Dawn-githup @anipenguin please just write down your previous work here, no need to mention your next work

**Comment by anipenguin** [comment]
### Previous Work
#### Marketing
- Spore Tweets and Announcement drafts  https://github.com/sporeprotocol/spore-scrum/issues/19#issuecomment-1911615619
- Reach out to Community Project: Titan Tiger https://github.com/sporeprotocol/spore-scrum/issues/19#issuecomment-1911648429

#### Docs
-  https://github.com/sporeprotocol/spore-contract/pull/56
-  https://github.com/sporeprotocol/spore-docs/pull/51
-  https://github.com/sporeprotocol/spore-docs/pull/56

_ClusterAgent & ClusterProxy related work_
-  https://github.com/sporeprotocol/spore-docs/pull/54
-  https://github.com/sporeprotocol/spore-docs/pull/55
-  Understanding relevant background and APIs

**Comment by ShookLyngs** [comment]
Done:
- spore-sdk
  - https://github.com/sporeprotocol/spore-sdk/pull/77
  - https://github.com/sporeprotocol/spore-sdk/pull/78
  - https://github.com/sporeprotocol/spore-sdk/pull/79
  - https://github.com/sporeprotocol/spore-sdk/pull/81
  - https://github.com/sporeprotocol/spore-sdk/pull/82
- sub-projects updates due to breaking changes in the spore-sdk
  - https://github.com/sporeprotocol/spore-scrum/issues/18

Left over:
- I think version bump is not urgent for this issue only
  - https://github.com/sporeprotocol/spore-sdk/pull/85
- discussion: should effort be put into this issue at the moment?
  - https://github.com/sporeprotocol/spore-graphql/issues/11

**Reply by ashuralyk** [reply]
> should effort be put into this issue at the moment

it got priority, if you have time, you could push a little bit

**Comment by Dawn-githup** [comment]
Done:
  
 - spore-sdk
   * spore-sdk mainnet test https://github.com/sporeprotocol/spore-sdk/pull/79
 - spore-demo
   * Fixed and verified some problems in spore-demo https://github.com/sporeprotocol/spore-demo/issues/54#issuecomment-1898207033
   * spore-demo adapts to the latest sdk/graphql test https://github.com/sporeprotocol/spore-demo/pull/64
 - spore-graphql
   * Verification based on new version of spore-graphql https://github.com/sporeprotocol/spore-demo/issues/54#issuecomment-1898207033
 - PhilosopherStone
   * PhilosopherStone testing/issue resolution verification. https://github.com/lee920217/PhilosopherStone/issues?q=is%3Aissue+author%3ADawn-githup+is%3Aclosed
  
Left over:
  1. PhilosopherStone functional modification test https://github.com/lee920217/PhilosopherStone/pull/59
  > > PhilosopherStone mainnet adaptation verification
  >
  >  I will do this later if I have time.
